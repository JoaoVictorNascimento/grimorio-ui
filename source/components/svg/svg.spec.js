import '../../../internals/test/helper';

import Svg from './index';

/** @test {Svg} */
describe('Button component', () => {
/** @test {Svg#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Svg />
      );
      expect(wrapper.length).to.equal(1);
    });
  });
});
