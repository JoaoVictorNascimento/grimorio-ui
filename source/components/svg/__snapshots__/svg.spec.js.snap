// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component render correctly 1`] = `"<Svg width={150} height={50} src=\\"mock\\" align=\\"middle\\" />"`;
